frontend/src/components/Table/Table.tsx

```diff
@@ -305,14 +307,19 @@ const Table = <T extends object>(props: PropsWithChildren<TableProps<T>>): React
               renderExpandRowStateButton(
                 props.detailsPanel && props.detailsPanel.checkExpanded(row.original, expandedRows),
                 'td',
-                () => handleExpandClick(row.original),
+                e => handleExpandClick(e, row.original),
               )}
+            {props.canRowExpand && !props.canRowExpand(row) ? (
```

Comment:
Fixes issue where the column is not included and all buildings are incorrectly aligned.